CONTRIBUTING.md

```diff
@@ -156,8 +179,23 @@ rule is that they should be as concise as possible.  Specifically:
   names, or methods and fields within a single class;
 * use a single blank line between top-level class definitions, or none
   if the classes are very small;
-* do not use docstrings;
-* for arguments with a type and a default, use spaces around the `=`.
+* do not use docstrings.
+
+Stub files should only contain information necessary for the type
+checker, and leave out unnecessary detail:
+* for arguments with a default, use `...` instead of the actual
+  default;
+* for arguments with a type and a default, use spaces around the `=`;
```

Comment:
I think this belongs better in the previous list, since it's purely about style and not about information content.

Reply:
That is a good point. I grouped it back in another commit.